### DISCUSSION TITLE
Nylo Network request data caching

### DISCUSSION THREAD

**Discussion by jitendravn** [nylo-core/nylo#165] [Ideas] [answered]
Tanstack query has caching feature on query request
can we have the same in Nylo 
so we don't need to call network request again and again

**Comment by agordn52** [comment, marked as answer]
Hi @jitendravn,

I like this idea! 
I'll add it into Nylo v6

**Reply by jitendravn** [reply]
Currently, I'm using fl_query for that.
there is one drawback I have seen 
assume
you update any data from your backend panel
but because of caching
its updating 
when you come to this screen
@agordn52
